src/cascadia/TerminalSettingsEditor/ColorSchemes.cpp

```diff
@@ -89,6 +89,16 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         const auto colorScheme{ args.AddedItems().GetAt(0).try_as<Model::ColorScheme>() };
         CurrentColorScheme(colorScheme);
         _UpdateColorTable(colorScheme);
+
+        // Set the text disclaimer for the text box
+        hstring disclaimer{};
+        const std::wstring schemeName{ colorScheme.Name() };
+        if (InBoxSchemes.find(schemeName) != InBoxSchemes.end())
+        {
+            // load disclaimer for in-box profiles
+            disclaimer = RS_(L"ColorScheme_DeleteButtonDisclaimerInBox");
+        }
```

Comment:
why is this not bound to a visibility like the Base Layer one is?

Reply:
Or, why is this not bound to `CanDeleteCurrentScheme`?

Reply:
I could simplify this down to just toggling visibility, but I'm wondering if we'll have other disclaimers here. Particularly thinking about Proto-Extensions. We could just add an if-statement here to detect that a scheme came from a proto-extension and load a different disclaimer for it.